### PR TITLE
feat(app): Add holographic material to cientos

### PR DESCRIPTION
**Pull request by JaimeTorrealba**

### CODE REVIEW

**Comment by andretchen0** [comment]
@JaimeTorrealba 

Sounds cool! Will review tomorrow.

File: docs/guide/materials/holographic-material.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+| **scanlineSize**       | Controls the size of the scanlines. Ranges from 1 to 15.            | `Number`                                            | 8.0       |
+| **hologramBrightness** | Controls the brightness of the hologram. Ranges from 0.0 to 2.0.    | `Number`                                            | 1.2       |
+| **signalSpeed**        | Controls the speed of the signal effect. Ranges from 0.0 to 2.0.    | `Number`                                            | 0.45      |
+| **hologramColor**      | Specifies the color of the hologram.                                | `String`                                            | "#00d5ff" |
+| **enableBlinking**     | Enables or disables the blinking effect.                            | `Boolean`                                           | true      |
+| **hologramOpacity**    | Specifies the opacity of the hologram.                              | `Number`                                            | 1.0       |
+| **enableBlinking**     | Enables or disables the blinking effect.                            | `Boolean`                                           | true      |
```
Comment (andretchen0) [inline docs/guide/materials/holographic-material.md:52]: This line can be deleted. It was added to the table twice.

File: docs/guide/materials/holographic-material.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <TresPerspectiveCamera :position="[3, 3, 3]" />
+    <Sphere :scale="0.5">
+      <HolographicMaterial />
+    </Sphere>
+    <TresAmbientLight />
+  </TresCanvas>
+</template>
+```
+
+### You can also replace the material of an existing mesh like this:
```
Comment (andretchen0) [inline docs/guide/materials/holographic-material.md:36]: (non-blocking)

To me, it looks a little funny to have this long header floating between 2 code blocks. Maybe change to:

````
### Replace a mesh's material

You can also replace the material of an existing mesh like this:
````

**Review by andretchen0 — CHANGES_REQUESTED** [review]
Hey @JaimeTorrealba !

Looks great! Nice addition!

Just a few changes.

File: src/core/materials/holographicMaterial/material.ts [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+  Color,
+  NormalBlending,
+  AdditiveBlending,
+  FrontSide,
+  BackSide,
+  DoubleSide,
+} from 'three'
+import type { Side, Blending } from 'three'
+
+// Original author Anderson Mancini - https://github.com/ektogamat/threejs-vanilla-holographic-material
```
Comment (andretchen0) [inline src/core/materials/holographicMaterial/material.ts:17]: @JaimeTorrealba @alvarosabu 

Is this sufficient for the license? 

I have zero legal knowledge, but reading over [the MIT license from the repo](https://github.com/ektogamat/threejs-vanilla-holographic-material/blob/main/LICENSE), it sounds like the whole text needs to accompany Anderson Mancini's code.

Here's the salient bit:

> The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

**Comment by JaimeTorrealba** [comment]
I already implemented all the feedbacks, as always thanks for take your time to do it, and the quality of them, it makes me grow as a professional and have a better library.

Btw the **levioso** advice it was just perfect 👍

**Review by andretchen0 — APPROVED** [review]

**Comment by andretchen0** [comment]
> I already implemented all the feedbacks, as always thanks for take your time to do it, and the quality of them, it makes me grow as a professional and have a better library.

That's really nice to hear. Thanks!

**Pull request merged**